clinica/pipelines/t1_volume_register_dartel/t1_volume_register_dartel_pipeline.py

```diff
@@ -245,10 +220,10 @@ def build_core_nodes(self):
         dartel_existing_template = npe.MapNode(utils.DARTELExistingTemplate(),
                                                name='dartel_existing_template',
                                                iterfield=['image_files'])
-        if self.parameters['optimization_parameters'] is not None:
-            dartel_existing_template.inputs.optimization_parameters = self.parameters['optimization_parameters']
-        if self.parameters['regularization_form'] is not None:
-            dartel_existing_template.inputs.regularization_form = self.parameters['regularization_form']
+        # if self.parameters['optimization_parameters'] is not None:
+        #     dartel_existing_template.inputs.optimization_parameters = self.parameters['optimization_parameters']
+        # if self.parameters['regularization_form'] is not None:
+        #     dartel_existing_template.inputs.regularization_form = self.parameters['regularization_form']
```

Comment:
Why commenting these lines ?

Reply:
I should remove them actually. If you look at the current `dev` version of Clinica, you will notice that `'optimization_parameters'`/`'regularization_form'` have always `None` as default value and they are not set in the CLI.

Reply:
I finally removed it.